newIDE/app/src/AssetStore/AssetStoreContext.js

```diff
@@ -148,6 +149,7 @@ export const AssetStoreStateProvider = ({
   const [assetShortHeadersById, setAssetShortHeadersById] = React.useState<?{
     [string]: AssetShortHeader,
   }>(null);
+  const { ownedAssetShortHeaders } = React.useContext(AuthenticatedUserContext);
```

Comment:
Should this be called ownedPrivateAssetShortHeaders?
(open question, I'm not sure it's better)